app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventPresenter.java

```diff
@@ -116,6 +117,18 @@ private void listenChanges() {
             .subscribe(copyrightModelChange -> loadCopyright(false), Logger::logError);
     }
 
+    public String getShareableInformation() {
+        String doubleLineBreak = "\n\n";
+        StringBuilder data = new StringBuilder(event.getName() + doubleLineBreak + "Starts: " + DateUtils.formatDateWithDefault(DateUtils.FORMAT_DAY_COMPLETE, event.getStartsAt()) +
+            doubleLineBreak + "Ends at: " + DateUtils.formatDateWithDefault(DateUtils.FORMAT_DAY_COMPLETE, event.getEndsAt()));
+
```

Comment:
This is as good as not using StringBuilder. This is still concatenating

Reply:
I've removed concatenation now. Please see if it's fine.